pkgs/tools/typesetting/rubber/default.nix

```diff
@@ -1,23 +1,34 @@
 { fetchurl, stdenv, python2Packages, texinfo }:
 
-stdenv.mkDerivation rec {
-  name = "rubber-1.3";
+let
+  pypkgs = python2Packages;
```

Comment:
Please use either `python2Packages` or `pythonPackages` instead of a custom variable.